x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/multiple_conditions.cy.ts

```diff
@@ -0,0 +1,124 @@
+  addExceptionEntryFieldValueValue,
+  addExceptionFlyoutItemName,
+  submitNewExceptionItem,
+} from '../../../tasks/exceptions';
+import {
+  ADD_AND_BTN,
+  ADD_OR_BTN,
+  EXCEPTION_CARD_ITEM_NAME,
+  EXCEPTION_CARD_ITEM_CONDITIONS,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+} from '../../../screens/exceptions';
+
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+
+describe(
+  'Add multiple conditions and validate the generated exceptions',
+  { testIsolation: false },
+  () => {
+    before(() => {
+      esArchiverResetKibana();
+      login();
+      deleteAlertsAndRules();
+      // At least create Rule with exceptions_list to be able to view created exceptions
+      createRule({
+        ...getNewRule(),
+        query: 'agent.name:*',
+        index: ['exceptions*'],
+        exceptions_list: [],
+        rule_id: '2',
+      });
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+      goToExceptionsTab();
+    });
+
+    after(() => {
+      esArchiverUnload('exceptions');
+    });
+    const exceptionName = 'My item name';
+
+    it('Use multipe AND conditions and validate it generates one exception', () => {
+      // open add exception modal
+      openExceptionFlyoutFromEmptyViewerPrompt();
+
+      // add exception item name
+      addExceptionFlyoutItemName(exceptionName);
+
+      addExceptionEntryFieldValue('agent.name', 0);
+      addExceptionEntryFieldValueValue('foo', 0);
+
+      cy.get(ADD_AND_BTN).click();
+
+      addExceptionEntryFieldValue('@timestamp', 1);
+      addExceptionEntryFieldValueValue('123', 1);
+
+      submitNewExceptionItem();
+
+      // Only one Exception should generated
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 1);
+
+      // validate the And operator is displayed correctly
+      cy.get(EXCEPTION_CARD_ITEM_NAME).should('have.text', exceptionName);
+      cy.get(EXCEPTION_CARD_ITEM_CONDITIONS).should(
+        'have.text',
+        ' agent.nameIS fooAND @timestampIS 123'
+      );
+      removeException();
+    });
+
+    it('Use multipe OR conditions and validate it generates multiple exceptions', () => {
+      // open add exception modal
+      openExceptionFlyoutFromEmptyViewerPrompt();
+
+      // add exception item name
+      addExceptionFlyoutItemName(exceptionName);
+
+      // exception item 1
+
+      addExceptionEntryFieldValue('agent.name', 0);
+      addExceptionEntryFieldValueValue('foo', 0);
+
+      cy.get(ADD_OR_BTN).click();
+
+      addExceptionEntryFieldValue('@timestamp', 1);
+      addExceptionEntryFieldValueValue('123', 1);
+
+      submitNewExceptionItem();
+
+      // Two Exceptions should be generated
+      cy.get(EXCEPTION_ITEM_VIEWER_CONTAINER).should('have.length', 2);
+
+      // validate the details of the first exception
+      cy.get(EXCEPTION_CARD_ITEM_NAME).eq(0).should('have.text', exceptionName);
+
+      cy.get(EXCEPTION_ITEM_ACTIONS_BUTTON).eq(1).click();
```

Comment:
Why are we finishing the test with actions?

Reply:
This is not an action, I am using the `cy.get(EXCEPTION_ITEM_ACTIONS_BUTTON).eq(0).click(); cy.get(REMOVE_EXCEPTION_BTN).click();` to remove the generated Exceptions

Reply:
Let's sync to discuss different things of improving this :)

Reply:
Removed it since we agreed to use the ` esArchiverResetKibana`